stable_baselines/bench/monitor.py

```diff
@@ -15,22 +15,21 @@
 class Monitor(gym.Wrapper):
     EXT = "monitor.csv"
     file_handler = None
+    """
```

Comment:
Why is the docstring here? I can see a case for it being a class docstring (although I think it was fine as the constructor docstring), but in the current placement it looks like a docstring for `file_handler`

Reply:
Good point, I moved it up ;) (I did that at first because sphinx auto-doc was not working with the previous state)